paddle/fluid/framework/new_executor/new_executor_defs.h

```diff
@@ -363,6 +367,7 @@ class Instruction {
   std::shared_ptr<ExecutionContext> execution_ctx_;
 
   std::vector<size_t> gc_check_var_list_;
+  std::vector<size_t> need_stream_sync_var_list_;
```

Comment:
Consider the operations performed on these vars, `need_record_stream_vars` is more suitable

Reply:
This code has been deleted, please check the new commit.